.github/workflows/callable-scan.yaml

```diff
@@ -67,9 +67,10 @@ jobs:
         run: |
           PACKAGES=""
           PACKAGE_NAME=$(echo "$BASE_REPO" | awk -F'/' 'BEGIN{OFS="/"} {if (NF > 2) {for (i=3; i<=NF; i++) printf (i==3 ? $i : "/"$i); print ""} else print ""}')/$(yq e '.metadata.name' zarf.yaml)
+          PACKAGE_NAME_ENCODED=$(echo "$PACKAGE_NAME" | sed 's/\//%2F/g')
 
           # Check for the public package
-          if gh api --silent /orgs/"$REPO_OWNER"/packages/container/"$PACKAGE_NAME" >/dev/null 2>&1; then
+          if gh api --silent /orgs/"$REPO_OWNER"/packages/container/"$PACKAGE_NAME_ENCODED" >/dev/null 2>&1; then
```

Comment:
Can I request that these variables be enclosed in curly braces while we're here changing them?

Reply:
fixed